### DISCUSSION TITLE
CSE Course Registration

### DISCUSSION THREAD

**Discussion by sree-sushma** [snigi-gupta/AttendingUniversityAtBuffalo#61] [Q&A]
Hi Snigdha,
Thank you so much for taking time to help us.
I have a doubt regarding course registration, there is a big list of seminars and different professors for every seminar and most of them are already closed. This semester I am planning to take 5 courses with a seminar included. So I would like to register for a seminar which is really light in load. Can you help me with this, on what basis I can select, or if possible u can suggest one.
Or is there any alternative for seminars like independent study or something else which is really light on load.

**Comment by snigi-gupta** [comment]
@sree-sushma 

I am not sure if this is still helpful to you, but regardless here is my take.

Why do you want to take 5 courses? To save money $3000? If that is the only reason then please don't. Your learning might get hampered and the workload is too much. You can always drop the course if you don't feel like continuing and that should be fine. But still, don't just pick 5 courses to be just done with it. Make sure you learn first.

If you are taking 5 courses coz you can and handle the pressure then definitely go ahead with it!
Kneply's and Wei seminar are light. I took these so I know only about those. 

In general if I were you I would audit the seminars and talk to the current senior batch for these details. Since I graduated a lot might have changed, so stay in touch with your current seniors and seek their guidance!

Sorry for the late reply (I was on a tour)
Please show your support by following me on https://www.instagram.com/yourtechdiary/
